spec/book_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../book'
```

Comment:
[ OPTIONAL ] All specs are failing on my system. Please make sure that every test passes before merging your branch with the main.
![tests](https://user-images.githubusercontent.com/35504149/140584935-45d6fea3-f20c-472c-86fa-0dad7b3608b2.png)